consensus/state_processing/src/per_block_processing/verify_sync_contribution.rs

```diff
@@ -0,0 +1,114 @@
+use super::errors::{AttestationInvalid as Invalid, BlockOperationError};
```

Comment:
I assume this file is a scrap and will get deleted.

Reply:
This raises a good point.. with attestation processing we have verification steps as we insert attestations into the op pool, and verification steps as we retrieve from the op pool. Do we similar want some verification as we retrieve a sync aggregate from the op pool? This would be that piece of code, which I haven't implemented :slightly_smiling_face: